mw-webapp/src/component/image/Image.tsx

```diff
@@ -25,21 +27,75 @@ interface ImageProps {
    * Data attribute for cypress testing
    */
   dataCy?: string;
+
+  /**
+   * Enlarge Image
+   */
+  isZoomable?: boolean;
+
+  /**
+   * Enlarge Image Styles
+   */
+  enlargeClassName?: string;
+
+  /**
+   * Enlarge Image Styles
+   */
+  isDisableZoom?: boolean;
 }
 
 /**
  * Сomponent for displaying images
  */
 export const Image = (props: ImageProps) => {
-  const className = clsx(styles.image, props.className);
+  const {src, alt, dataCy, className, enlargeClassName, isZoomable, isDisableZoom} = props;
 
-  return (
+  const [zoomable, setZoomable] = useState(isZoomable);
+
+  useEffect(() => {
+    if (isDisableZoom) {
+      setZoomable(false);
+    } else {
+      setZoomable(isZoomable);
+    }
+  }, [isDisableZoom, isZoomable]);
+
+  const combinedClassName = clsx(styles.image, className);
+  const combinedEnlargeClassName = clsx(styles.image, enlargeClassName);
```

Comment:
too much classes, clsx could combine 2 and more classes with no limits this wat ```clsx(styles.1, ["styles.2"]: true.... or other ways)```, check clsx API and other components like Button

Reply:
All right